InternxtDesktop/AppDelegate.swift

```diff
@@ -52,8 +52,9 @@ class AppDelegate: NSObject, NSApplicationDelegate , PKPushRegistryDelegate {
     
     var listenToLoggedIn: AnyCancellable?
     var refreshTokensTimer: AnyCancellable?
+    var signalEnumeratorTimer: AnyCancellable?
+    var storageDebouncer = Debouncer(delay: 15.0)
```

Comment:
`storageDebouncer` doesn't sound like a good name to me, I feel like this doesn't communicate the purpose of this, some alternatives:

- `usageUpdateDebouncer`
- `usageRefreshDebouncer`